docroot/themes/custom/uids_base/templates/field/field--node--field-event-registration-link--event--default.html.twig

```diff
@@ -0,0 +1,45 @@
+{#
+/**
+ * @file
+ * Theme override for a field.
+ *
+ * To override output, copy the "field.html.twig" from the templates directory
+ * to your theme's directory and customize it, just like customizing other
+ * Drupal templates such as page.html.twig or node.html.twig.
+ *
+ * Instead of overriding the theming for all fields, you can also just override
+ * theming for a subset of fields using
+ * @link themeable Theme hook suggestions. @endlink For example,
+ * here are some theme hook suggestions that can be used for a field_foo field
+ * on an article node type:
+ * - field--node--field-foo--article.html.twig
+ * - field--node--field-foo.html.twig
+ * - field--node--article.html.twig
+ * - field--field-foo.html.twig
+ * - field--text-with-summary.html.twig
+ * - field.html.twig
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - multiple: TRUE if a field can contain multiple items.
+ * - items: List of all the field items. Each item contains:
+ *   - attributes: List of HTML attributes for each item.
+ *   - content: The field item's content.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ *
+ * @see template_preprocess_field()
+ */
+#}
+
+{% for item in items %}
+  <a href="{{ item.content['#url'] }}" title="{{ item.content['#title'] }}" role="button" class="bttn bttn--full bttn--primary">
```

Comment:
Why add the title attribute but then hardcode the text?

`<a href="{{ item.content['#url'] }}" class="bttn bttn--full bttn--primary">{{ item.content['#title']|render }}</a>` - for reference docroot/sites/grad.admissions.uiowa.edu/modules/grad_admissions_core/templates/field--node--field-area-of-study-apply-link--area-of-study.html.twig

Reply:
I changed the title parameter for the <a> tag to match the hardcoded value. The title was empty anyways, which would have left the parameter value as empty, but good catch!

Reply:
is the field collecting a title or is it link only?

Reply:
link only, title is hidden from the form. Didn't want the user to get "creative" with the what the link button title.